Array.java

```diff
@@ -0,0 +1,38 @@
+/* package whatever; // don't place package name! */
+
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+import java.util.Scanner;
+
+
+/* Name of the class has to be "Main" only if the class is public. */
+class Ideone
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+	    
+	    int m =5;
+		int ar[] = {1,2,3,4,5};
+		int temp;
```

Comment:
can we refactor this into a function to rotate the first element to back of the array ?